service/matching/physical_task_queue_manager_test.go

```diff
@@ -91,9 +91,16 @@ func (s *PhysicalTaskQueueManagerTestSuite) SetupTest() {
 	s.NoError(err)
 	engine.partitions[prtn.Key()] = prtnMgr
 
+	if s.fairness {
+		prtnMgr.config.NewMatcher = true
+		prtnMgr.config.EnableFairness = true
+	} else if s.newMatcher {
+		prtnMgr.config.NewMatcher = true
+	}
```

Comment:
hmm, the stuff on line 70-74 doesn't take care of this? if we do need this, maybe we don't need that?

reading this whole function again... it duplicates a lot of real code to set things up. I wish we could consolidate. but probably not now.

Reply:
I still don't understand why this change was needed. maybe we can huddle on it.